README.md

```diff
@@ -1,623 +1,1519 @@
+curl -L -O https://github.com/ELEVATE-Project/samiksha-service/raw/main/documentation/1.0.0/dockerized/envs/notification_env
 
--O https://github.com/ELEVATE-Project/samiksha-service/raw/main/documentation/1.0.0/dockerized/envs/user_env \
+curl -L -O https://github.com/ELEVATE-Project/samiksha-service/raw/main/documentation/1.0.0/dockerized/envs/scheduler_env
 
--O https://github.com/ELEVATE-Project/samiksha-service/raw/main/documentation/1.0.0/dockerized/envs/env.js
+curl -L -O https://github.com/ELEVATE-Project/samiksha-service/raw/main/documentation/1.0.0/dockerized/envs/user_env
 
+curl -L -O https://github.com/ELEVATE-Project/samiksha-service/raw/main/documentation/1.0.0/dockerized/envs/env.js
 ```
+
 >  **Note:** Modify the environment files as necessary for your deployment using any text editor, ensuring that the values are appropriate for your environment. The default values provided in the current files are functional and serve as a good starting point. Refer to the sample env files provided at the [Samiksha](https://github.com/ELEVATE-Project/samiksha-service/blob/main/.env.sample), [User](https://github.com/ELEVATE-Project/user/blob/master/src/.env.sample), [Notification](https://github.com/ELEVATE-Project/notification/blob/master/src/.env.sample), [Scheduler](https://github.com/ELEVATE-Project/scheduler/blob/master/src/.env.sample), [Interface](https://github.com/ELEVATE-Project/interface-service/blob/main/src/.env.sample) and [Entity-management](https://github.com/ELEVATE-Project/entity-management/blob/main/src/.env.sample) repositories for reference.
 
   
 
+  
+
 >  **Caution:** While the default values in the downloaded environment files enable the Project Application to operate, certain features may not function correctly or could be impaired unless the adopter-specific environment variables are properly configured.
 
   
 
+  
+
 4.  **Download `replace_volume_path` Script File**
 
   
 
+  
+
 -  **Ubuntu/Linux/Mac**
 
-```
+  
 
+```
 curl -OJL https://raw.githubusercontent.com/ELEVATE-Project/samiksha-service/refs/heads/main/documentation/1.0.0/dockerized/scripts/mac-linux/replace_volume_path.sh
-
 ```
-<!--
-
-- **Windows**
-
+-  **Windows**
 ```
-
 curl -OJL https://raw.githubusercontent.com/ELEVATE-Project/samiksha-service/refs/heads/main/documentation/1.0.0/dockerized/scripts/windows/replace_volume_path.bat
-
 ```
-
--->
+5.  **Run `replace_volume_path` Script File**
 
   
 
-5.  **Run `replace_volume_path` Script File**
-
   
 
 -  **Ubuntu/Linux/Mac**
 
+  
+
 1. Make the `replace_volume_path.sh` file an executable.
 
-```
+  
 
+```
 chmod +x replace_volume_path.sh
-
 ```
 
+  
+
 2. Run the script file using the following command.
 
-```
+  
 
+```
 ./replace_volume_path.sh
-
 ```
 
   
-  
-  
+-  **Windows**
+  1. Run the script file using the following command.
+  ```
+replace_volume_path.sh
+```
 
 6. ** Download `docker-compose-up` & `docker-compose-down` Script Files **
 
   
 
+  
+
 -  **Ubuntu/Linux/Mac**
```

Comment:
@borkarsaish65  its a subheading correct the alignment

Reply:
corrected